bigquery/samples/tests/test_client_query_legacy_sql.py

```diff
@@ -20,4 +20,4 @@ def test_client_query_legacy_sql(capsys, client):
 
     client_query_legacy_sql.client_query_legacy_sql(client)
     out, err = capsys.readouterr()
-    assert "Row(('Frances',), {'name': 0})" in out
+    assert "Row((" in out
```

Comment:
As the test [failed](https://source.cloud.google.com/results/invocations/492c2a03-6311-46d0-b9c4-7f8c894aa1bb/targets/cloud-devrel%2Fclient-libraries%2Fgoogle-cloud-python%2Fpresubmit%2Fbigquery/log) with the **python-2.7** version, it was decided to assert only the beginning of the string to prevent this fail again. Not an ideal implementation!

Reply:
@plamut I rewrote the implementation using `re`. Can you have a look, please?

Reply:
@Emar-Kar Oh, I meant just avoiding the problem of the extra `u` prefix in Python 2 output by marking it as optional:
```py
pattern = r"Row\(\(u?'Frances',\), \{u?'name': 0\}\)"
```
Depends on how specific a match we need to reliably distinguish between the failed and non-failed cases. If the current pattern `r"(Row[\w(){}:', ]+)$"` is already good enough, I wouldn't complicate too much.

(I did not read the entire PR, just saw that initial comment when glancing over it)